modules/end-user-guide/pages/url-parameter-for-container-image.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: CONCEPT
+:description: URL parameter for a container image
+:keywords: parameter, URL, container, image
+:navtitle: URL parameter for a container image
+//:page-aliases:
+
+[id="url-parameter-for-container-image"]
+= URL parameter for a container image
+
+You can use the `image` parameter to add a path to a container image in the following scenarios:
+
+* The Git repository contains no devfile, and you want to start a new workspace with this container.
+
+* The Git repository contains a devfile, and you want this container image to run instead of the first `container` `image` listed in the `components` section of the devfile.
+
+The URL parameter for the path to a container image is `image=`:
+
+[source,subs="+quotes,+attributes,+macros"]
+----
+pass:c,a,q[{prod-url}]#__<git_repository_url>__?image=__<container_image_url>__
```

Comment:
I do not think it is URL actually, but rather image reference e.g. docker.io/repo/test-image:tag

Reply:
would be great to have an example as part of the section

Reply:
@ibuziuk I did this ficha.
Issue: https://github.com/eclipse/che/issues/21990
PR: https://github.com/eclipse-che/che-dashboard/pull/761
It is an image reference e.g. docker.io/repo/test-image:tag.


```
https://che-dogfooding.apps.che-dev.x6e0.p1.openshiftapps.com#https://github.com/eclipse-che/che-dashboard.git?image=quay.io/eclipse/che-plugin-registry:next

```